content/101/authenticating-cli-tools.md

```diff
@@ -38,5 +38,4 @@ list of variables and what their corresponding values should be:
  email address)
  - `OS_PASSWORD` - Your Cloud A password
  - `OS_TENANT_NAME` - Typically the same as OS_USERNAME
- - `OS_AUTH_URL` - Set to http://api.cell01.clouda.ca:5000/v2.0/ Cloud A
- Windows Environment
+ - `OS_AUTH_URL` - Set to https://keystone.ca-ns-1.clouda.ca:8443/v2.0
```

Comment:
This is the auth URL for `ca-ns-1`. Would it be better to put `<region-name>` or similar in the example URL?